tests/py2puml/test_py2puml.py

```diff
@@ -1,5 +1,47 @@
-from py2puml import __version__
+from py2puml import __version__, py2puml
 
 
 def test_version():
-    assert __version__ == '0.3.0'
+    assert __version__ == '0.3.1'
+
+
+def test_py2puml():
+    """test py2puml on py2puml/domain."""
+    out = py2puml.py2puml('py2puml/domain', 'py2puml.domain')
```

Comment:
can you move this line below the expected output, so that the actual output and the assertion are close to each other (for readability sake), please?

And also, please rename the `out` variable into something more meaningful: `py2puml_domain` for example.

Reply:
Yes, i updated test case.